shims/common/src/main/scala/io/glutenproject/GlutenConfig.scala

```diff
@@ -752,7 +752,10 @@ object GlutenConfig {
       .doc(
         "Whether to spill the partition buffers when buffers are full. " +
           "If false, the partition buffers will be cached in memory first, " +
-          "and the cached buffers will be spilled when reach maximum memory.")
+          "and the cached buffers will be spilled when reach maximum memory." +
```

Comment:
Since it's not a configurable parameters, let's remove the flag and set it when local/remote shuffle is used internall.

Reply:
Looks like CH backend still uses this flag. @liuneng1994 Could you help to check if we still need this configuration for CH?

Reply:
> Looks like CH backend still uses this flag. @liuneng1994 Could you help to check if we still need this configuration for CH?

yes, ch need this configuration